src/components/SearchBar/SearchBar.tsx

```diff
@@ -0,0 +1,38 @@
+import {Dispatch, FormEvent, SetStateAction} from "react";
+import {useTranslation} from "react-i18next";
+import classNames from "classnames";
+import {ReactComponent as SearchIcon} from "assets/icons/search.svg";
+import {ReactComponent as ClearIcon} from "assets/icons/close.svg";
+import "./SearchBar.scss";
+
+type SearchBarProps = {
+  className?: string;
+  disabled?: boolean;
+  value: string;
+  handleValueChange: Dispatch<SetStateAction<string>>;
+};
+
+/*
+ * stateless search bar component.
+ * if the input is not empty, it's clearable using the X button
+ */
+export const SearchBar = (props: SearchBarProps) => {
```

Comment:
Should probably use debounce

Reply:
could make sense, but would need to be in the other PR since the state isn't used here for anything other than the input value itself